packages/strapi-admin/admin/src/components/EventInput/index.js

```diff
@@ -19,6 +19,8 @@ const EventInput = ({ onChange, name: inputName, value: inputValue }) => {
     media: ['media.create', 'media.update', 'media.delete'],
   };
 
+  const disabledEvents = ['media.update'];
```

Comment:
Can you add some comments here so we know why we need to remove it from the "authorised" ones.